src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="largeForm"
+      action="/"
+      method="post"
+      data-qa="big"
+    >
+      <img
+        class="firstImage"
+        src="./images/Search.svg"
+        alt="search"
+      >
+
+      <input
+        type="text"
+        class="largeInput"
+        placeholder="Try &quot;Los Angeles&quot"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form
+      class="smallForm"
+      action="/"
+      method="post"
+      data-qa="small"
+    >
+      <img
+        src="./images/Search.svg"
+        alt="search"
+        class="secondImage"
+      >
+      
+      <input
+        type="text"
+        class="smallInput"
+        placeholder="Try &quot;Los Angeles&quot"
+      >
```

Comment:
```suggestion
    <form
      class="form"
      action="/"
      method="post"
      data-qa="big"
    >
      <img
        class="image"
        src="./images/Search.svg"
        alt="search"
      >

      <input
        type="text"
        class="input"
        placeholder="Try &quot;Los Angeles&quot"
        data-qa="keypress"
      >
    </form>

    <form
      class="form form--small"
      action="/"
      method="post"
      data-qa="small"
    >
      <img
        src="./images/Search.svg"
        alt="search"
        class="image image--small"
      >
      
      <input
        type="text"
        class="input input--small"
        placeholder="Try &quot;Los Angeles&quot"
      >
```
lets make it work like this